integration-tests/start.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+type tokenBody struct {
+	Token string
+}
+
+func main() {
+
+	// Make a user POST request
+	msg := []byte(`{"email":"test@test.com", "password":"abcdef"}`)
+	postRs, postErr := http.Post("http://localhost:8180/users", "application/json", bytes.NewBuffer(msg))
```

Comment:
In go, it's preferable to use short and concise names for variables. For example, renaming `postErr` will avoid you to create the `tokenErr` below since you can reuse it.

```suggestion
	resp, err := http.Post("http://localhost:8180/users", "application/json", bytes.NewBuffer(msg))
```

Also, if you're using the `response` only to close the `body` stream, move `postRs.Body.Close()` to after error validation.

Reply:
Done.